src/bitmessageqt/bitmessageui.py

```diff
@@ -1,20 +1,24 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=too-many-locals,c-extension-no-member
+"""
+bitmessageui.py
+===============
 
-# Form implementation generated from reading ui file 'bitmessageui.ui'
-#
-# Created: Mon Mar 23 22:18:07 2015
-#      by: PyQt4 UI code generator 4.10.4
-#
-# WARNING! All changes made in this file will be lost!
+Originally generated from reading ui file 'bitmessageui.ui'. Since then maintained manually.
+"""
+
+import sys
 
 from PyQt4 import QtCore, QtGui
+
+from bitmessageqt import bitmessage_icons_rc  # pylint: disable=unused-import
```

Comment:
from bitmessageqt import settingsmixin, bitmessage_icons_rc  # pylint: disable=unused-import

Reply:
I was concerned there might have been side-effects from this import. After a closer look I don't think there is. Removed.